adyenofficial.php

```diff
@@ -958,7 +958,7 @@ public function displayForm()
                 'For configuration "ADYEN_APIKEY_LIVE" an exception was thrown: ' . $e->getMessage()
             );
         } catch (\Adyen\PrestaShop\exception\MissingDataException $e) {
-            $this->logger->debug('The configuration "ADYEN_APIKEY_LIVE" has no value set.');
+            $this->logger->error('The configuration "ADYEN_APIKEY_LIVE" has no value set.');
```

Comment:
this can be a warning as well, we set this as a required field so the config can not be saved if it's not filled

Reply:
Change committed